i18n/fr.toml

```diff
@@ -62,8 +58,8 @@ table_of_contents = "Table des matières"
 load_comments = "Afficher les commentaires"
 
 # Copy code block button.
-copied = "Copié!"
-copy_code_to_clipboard = "Copier le code dans le presse-papiers"
+copied = "Copié !"
```

Comment:
Is the space before the exclamation mark intentional?

Reply:
Yes, that's right. In French typography, the punctuation marks '!', '?' and ':' are always preceded by a non-breaking space.

Reply:
Interesting!